src/main/java/org/cryptomator/cryptofs/CryptoFileSystemProvider.java

```diff
@@ -161,18 +161,35 @@ public static void initialize(Path pathToVault, CryptoFileSystemProperties prope
 
 	/**
 	 * Checks if the folder represented by the given path exists and contains a valid vault structure.
+	 * <p>
+	 * See {@link DirStructure#VAULT} for the criteria of being a valid vault.
 	 *
-	 * @param pathToVault         A directory path
+	 * @param pathToAssumedVault A directory path
 	 * @param vaultConfigFilename Name of the vault config file
-	 * @param masterkeyFilename   Name of the masterkey file
+	 * @param masterkeyFilename Name of the masterkey file
 	 * @return <code>true</code> if the directory seems to contain a vault.
 	 * @since 2.0.0
 	 */
-	public static boolean containsVault(Path pathToVault, String vaultConfigFilename, String masterkeyFilename) {
-		Path vaultConfigPath = pathToVault.resolve(vaultConfigFilename);
-		Path masterkeyPath = pathToVault.resolve(masterkeyFilename);
-		Path dataDirPath = pathToVault.resolve(Constants.DATA_DIR_NAME);
-		return (Files.isReadable(vaultConfigPath) || Files.isReadable(masterkeyPath)) && Files.isDirectory(dataDirPath);
+	public static boolean containsVault(Path pathToAssumedVault, String vaultConfigFilename, String masterkeyFilename) {

```

Comment:
Let's use the opportunity to ditch this method. No need to retain it, we have already removed a lot of other deprecated methods in 2.0.0

Reply:
Done in c92a45c